sklearn/metrics/classification.py

```diff
@@ -2353,8 +2334,10 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
         Sample weights.
 
     pos_label : int or str, default=None
-        Label of the positive class. If None, the maximum label is used as
-        positive class
+        Label of the positive class.
+        When ``pos_label=None``, if y_true is in {-1, 1} or {0, 1},
```

Comment:
Defaults to the greater label unless y_true is all 0 or all -1 in which case pos_label defaults to 1.

How about that?